website/src/pages/v1/serve/features/monitoring-tracing/prometheus.mdx

```diff
@@ -4,83 +4,234 @@ description:
+| `operationName` | Configurable<br/>On by default | If available, the name of the GraphQL operation requested, otherwise `Anonymous`.<br/><br/>This can help you identify which operations are slow or failing.<br/><br/>We recommend you always provide an operation name to your queries and mutations to help performance analysis and bug tracking.                                                                                     |
+| `operationType` | Configurable<br/>On by default | The type of the GraphQL operation requested. It can be one of `query`, `mutation`, or `subscription`.<br/><br/>This can help you differentiate read and write performance of the system. It can for example help understand cache impact.                                                                                                                                               |
+
+#### Sample output
+
+```
+# HELP graphql_yoga_http_duration Time spent on HTTP connection
+# TYPE graphql_yoga_http_duration histogram
+graphql_yoga_http_duration_bucket{le="0.005",method="GET",statusCode="200",operationName="Anonymous"} 1
+graphql_yoga_http_duration_bucket{le="0.01",method="GET",statusCode="200",operationName="Anonymous"} 1
+graphql_yoga_http_duration_bucket{le="0.025",method="GET",statusCode="200",operationName="Anonymous"} 1
+graphql_yoga_http_duration_bucket{le="0.05",method="GET",statusCode="200",operationName="Anonymous"} 1
+graphql_yoga_http_duration_bucket{le="0.1",method="GET",statusCode="200",operationName="Anonymous"} 1
+graphql_yoga_http_duration_bucket{le="0.25",method="GET",statusCode="200",operationName="Anonymous"} 1
+graphql_yoga_http_duration_bucket{le="0.5",method="GET",statusCode="200",operationName="Anonymous"} 1
+graphql_yoga_http_duration_bucket{le="1",method="GET",statusCode="200",operationName="Anonymous"} 4
+graphql_yoga_http_duration_bucket{le="2.5",method="GET",statusCode="200",operationName="Anonymous"} 4
+graphql_yoga_http_duration_bucket{le="5",method="GET",statusCode="200",operationName="Anonymous"} 4
+graphql_yoga_http_duration_bucket{le="10",method="GET",statusCode="200",operationName="Anonymous"} 4
+graphql_yoga_http_duration_bucket{le="+Inf",method="GET",statusCode="200",operationName="Anonymous"} 4
+graphql_yoga_http_duration_sum{method="GET",statusCode="200",operationName="Anonymous"} 3
+graphql_yoga_http_duration_count{method="GET",statusCode="200",operationName="Anonymous"} 4
+```
+
+### `graphql_mesh_fetch_duration`
+
+This metric tracks the duration of outgoing (upstream) HTTP requests. It reports the time spent on
+each request made during the execution of the GraphQL operation as an histogram.
+
+This metric can provide insights into which upstream service is slowing down your queries.
+
+To enable this metric, set the `fetchMetrics` option to `true`.
```

Comment:
Does this mean requests to other GraphQL services? Maybe mention "subgraphs" here?

Reply:
How does this metric differ from `graphql_mesh_subgraph_execute_duration`?

Reply:
It means request to any HTTP upstream sources. It can be a subgraph, a REST API, and SOAP endpoint, whatever

Reply:
Ok, so the difference is that `graphql_mesh_fetch_duration` is mesuring any `fetch` call, while `graphql_mesh_subgraph_execute_duration` mesures the time to run the `executor` function of each subgraph. I will try to make this more clear